### ISSUE TITLE
Test|set up Gitpod to edit the EDAM repo online

### ISSUE THREAD

**Issue by matuskalas** [edamontology/edamontology#669]
This is a great very first issue for all newcomers and **everyone** who wants to edit the "source code" of EDAM, but without using Git locally. 🐙 (Outreachy applicants are welcome to contribute here! 🌞)

💻 Working with Gitpod requires a reasonably good internet connection, and a reasonably up-to-date web browser. **Optional: Please try** if it works on phones📱, on vintage hardware💾, or with unstable internet🕸, and write your experience! Million thanks🙏🏽

1. Fork this repository (upper right corner).
2. Please log in to Gitpod.io, launching your fork of the _edamontology_ repo there, by opening `https://gitpod.io/#https://github.com/<GitHub-username>/edamontology` in your web browser.
3. [Slightly advanced] Try if it's possible to set the [Git hooks](https://coderefinery.github.io/git-collaborative/05-hooks/) to work from the `.githooks` folder inside the _edamontology_ repository. Locally this is done by `git config core.hooksPath .githooks` in a command-line terminal, but I have **no idea** how to do it on Gitpod, because I haven't tried🤷🏽‍♀️. I assume it's possible!😉 (Steps for doing this locally in a terminal🖥 are documented in [CONTRIBUTING.md](https://github.com/edamontology/edamontology/blob/main/CONTRIBUTING.md#option-1)).
Irrespective of whether you succeed with Step 3., please continue with the following steps:
4. Make a dummy change in `EDAM_dev.owl`, and try to commit it to a **new, experimental Git branch** (see below if you need to learn or refresh Git: commits, branches). (If the Git hooks work, you should be able to see the output `Yay! Your commit to EDAM Ontology was successful.` somewhere.)
5. Now replace all `owl:Class` in the `EDAM_dev.owl` file with for example `wol:Class` 🦉, and try to commit that change. (If the Git hooks work, the commit should fail now (returning `0`), not allowing you to commit. But it's no disaster if the hooks don't work for you.)
7. [If you have NOT done this in a new, experimental branch], reset the HEAD of your current branch (_e.g._ `main`) back to the last "proper" commit. (This includes figuring out how to do that in Gitpod. Hints: Reverting a commit is NOT the solution; `git reset` is what to use in a command-line terminal.)
8. [If you HAVE done it in an experimental branch], please make sure that you don't continue doing your next, "real" changes into this branch.😉
9. Please share your experience, big thanks!🙏🏽 Does it work for you? Please post a comment to this issue if yes, thanks!😸
Does it not work for you? Please describe your problem as a comment to this issue and someone may help you!🆘

----

**If you need to learn or refresh Git, or test how to use Git in Gitpod, we recommend skimming through the following materials, and trying to do some of the hands-on exercises in Gitpod:**
 - Quick introduction|refresh of [Git and GitHub](https://coderefinery.github.io/git-refresher/02-refresher/) (101)
 - [Learn Git](https://coderefinery.github.io/git-intro/) (great for both beginners and advanced)
 - [Forking a repository and creating pull requests](https://coderefinery.github.io/git-collaborative/03-distributed/) on GitHub
 - Optional: Git hooks [quick overview](https://coderefinery.github.io/git-collaborative/05-hooks/)

----

When this works, you're ready to edit EDAM on Gitpod! 👍🏽 Your experience is very appreciated in the EDAM community, and will help other contributors! 🚀🙌🏽

**Comment by sarkar-ly** [comment]
I tried doing the first and second step but it doesn't grant me access@matus kalas

**Comment by matuskalas** [comment]
@sarkar-ly, have you tried with your fork https://gitpod.io/#https://github.com/sarkar-ly/edamontology, using your GitHub account to sign in to Gitpod?

**Comment by sarkar-ly** [comment]
Yes I did...I'll try again

**Comment by matuskalas** [comment]
😢 Hmm, I have no idea what the problem could be

**Comment by AnkitaxPriya** [comment]
@matuskalas In the 3rd step, do we need to add `post-commit` hook in the `.githooks` folder? I tried to do so but wasn't able to push the code. I have attached the screenshot for reference. 

![image](https://user-images.githubusercontent.com/44089458/116501805-68825280-a8cf-11eb-9232-d776e4aa05d6.png)

**Comment by AnkitaxPriya** [comment]
@matuskalas in the 4th task, I was able to commit a change but did not receive the message `Yay! Your commit to EDAM Ontology was successful.` I suppose this could probably be because I have been working on an older version of the `edamontology` repo which isn't updated with the `post-commit` hook. 

I have added the screenshot for reference. 

![image](https://user-images.githubusercontent.com/44089458/116502007-f65e3d80-a8cf-11eb-8d92-03e8401e01f9.png)

**Comment by AnkitaxPriya** [comment]
@matuskalas the 5th task was successful for me. I have attached the screenshot for reference here. 
![image](https://user-images.githubusercontent.com/44089458/116502276-b3509a00-a8d0-11eb-951f-85382f4d235e.png)

**Comment by sarkar-ly** [comment]
Did this URL work for you? https://gitpod.io/#https://github.com/<GitHub-username>/edamontology @ankitaxpriya

**Comment by AnkitaxPriya** [comment]
Yes @sarkar-ly it worked for me. In your case, use the link https://gitpod.io/#https://github.com/sarkar-ly/edamontology
Make sure you have forked the repo on your GitHub space already.

**Comment by AnkitaxPriya** [comment]
@matuskalas Do I need to create a separate issue to work on to "add a Gitpod configuration YAML file to the EDAM repository" or can I link the file to this issue only?

**Comment by sarkar-ly** [comment]
Thanks @ ankitaxpriya..I used the same URL but it didn't but I tried again and this time it worked

**Comment by sarkar-ly** [comment]
Committed the changes but receive the message as told by ankitaxpriya
![Screenshot (5)](https://user-images.githubusercontent.com/65134563/116583104-a0bb7c80-a933-11eb-8835-3799810dd5b1.png)
![Screenshot (6)](https://user-images.githubusercontent.com/65134563/116583186-b4ff7980-a933-11eb-8dd4-f336bb209e72.png)
attached the screenshot...is it okay or do we need changes

**Comment by rioualen** [comment]
Hi, 
Is this still an issue? If not, I'd suggest moving the content of the original post to the discussion or to the documentation or CONTRIBUTING.md file (as mentioned also in #670)
@matuskalas
